lizmap/modules/lizmap/classes/lizmapServices.class.php

```diff
@@ -282,14 +282,15 @@ public function __construct($readConfigPath, $globalConfig, $ldapEnabled, $varPa
             $this->allowUserAccountRequests = false;
         }
 
-        // set it for external requests, needed for file_get_contents
-        $userAgent = 'Lizmap';
+        $appInfos = \Jelix\Core\Infos\AppInfos::load();
```

Comment:
Retrieve the userAgent in an `else` statement of the statement ` if (isset($readConfigPath['services']['userAgent'])) {`. 
`AppInfos` loads xml file, and so it takes some resources. We could avoid to load this file if we don't need it.